SPECS/moby-containerd-cc/containerd.toml

```diff
@@ -0,0 +1,26 @@
+#root = "/var/lib/containerd"
+#state = "/run/containerd"
+#subreaper = true
+#oom_score = 0
+
+#[grpc]
+#  address = "/run/containerd/containerd.sock"
+#  uid = 0
+#  gid = 0
+
+#[debug]
+#  address = "/run/containerd/debug.sock"
+#  uid = 0
+#  gid = 0
+#  level = "info"
+
+version = 2
+[plugins]
```

Comment:
Double checking if containerd.toml == /etc/containerd/config.toml. If they are, could we include what is patched in https://dev.azure.com/mariner-org/mariner/_git/mariner-aks-pipelines?path=/patches/kata-img-plus.patch&version=GBdadelan/prototypes&line=519&lineEnd=519&lineStartColumn=1&lineEndColumn=57&lineStyle=plain&_a=contents here? So we no longer need to do that in agent baker. I think @ms-mahuber alluded this some time ago.

Reply:
I'm hesitant to include anything related to kata in this package as this is a separate package from kata. I think we want to keep using agentbaker at least for now. @ms-mahuber any thoughts?

Reply:
Either to keep using AgentBaker or to revisit the thought on amending the containerd config with kata-containers, kata-containers-cc package installation. We would ideally have a clean way of adding to (and removing from) the plugins sections on package installation (and uninstallation) time.

One thought we previously had, potentially discarded though, was to use the imports section of the containerd config (https://github.com/containerd/containerd/blob/main/docs/man/containerd-config.toml.5.md). We could set/unset a simple import and add/to in the additional config file - this could be safer in terms of not damaging the config.

Reply:
As discussed offline, this will most likely be done in AgentBaker instead of this RPM.